### ISSUE TITLE
Running the App

### ISSUE THREAD

**Issue by Prakruthi230695** [swpease/codenames#1]
Hey, I was trying to incorporate TypeScript into React by using an already existing app as a building block. I would like to know how do I run this app locally?

Thank you for your time!

**Comment by swpease** [comment]
I think that all you need to do is run `node server.js` from the project root and navigate to `localhost:3001/<string>`. [ref](https://expressjs.com/en/starter/hello-world.html)

**Comment by Prakruthi230695** [comment]
Thank you for your response! 

Yes, I did that. Tried the following:

`npm install`

`npm start`

It executes and displays the following:

```
> codenamesBackend@0.1.0 start
> nodemon ./server.js

[nodemon] 2.0.6
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,json
[nodemon] starting `node ./server.js`
listening on 3001

```

But it displays a blank page on port `3001`.
So, wanted to know if I missed anything else in addition to the above commands in order to get it running.

In the client directory, the `package.json` has a dependency on `react-scripts-ts` for the start script. So does that need to be configured?
